nixos/tests/kexec.nix

```diff
@@ -18,28 +16,32 @@ import ./make-test-python.nix ({ pkgs, lib, ... }: {
 
     node2 = { modulesPath, ... }: {
       virtualisation.vlans = [ ];
+      environment.systemPackages = [ pkgs.hello ];
       imports = [
         "${modulesPath}/installer/kexec/kexec-boot.nix"
+        "${modulesPath}/profiles/minimal.nix
       ];
     };
   };
 
   testScript = { nodes, ... }: ''
+    # Test whether reboot via kexec works.
     node1.wait_for_unit("multi-user.target")
     node1.succeed('kexec --load /run/current-system/kernel --initrd /run/current-system/initrd --command-line "$(</proc/cmdline)"')
     node1.execute("systemctl kexec >&2 &", check_return=False)
     node1.connected = False
     node1.connect()
     node1.wait_for_unit("multi-user.target")
 
-    # Check the machine with kexec-boot.nix profile boots up
+    # Check if the machine with kexec-boot.nix profile boots up
```

Comment:
```suggestion
    # Check if the machine with kexec-boot.nix profile boots up - that it complements the boot config, rather than replacing it
```

Reply:
I do not really understand that comment.

Reply:
Added _regular_. If it's not helpful, feel free to ignore it.

Reply:
not sure whats the `regular` config is? node1? since we just bootup node2 and shut it down again if it actually works. not sure what we would be replacing

Reply:
I imagined a situation where a kexec config would be mutually exclusive with other boot loading configs in the same NixOS configuration. It's what might make `node2` non-bootable, but perhaps we don't have to spell that out.